params/config.go

```diff
@@ -57,21 +57,21 @@ var (
 	// MainnetChainConfig is the chain parameters to run a node on the main network.
 	MainnetChainConfig = &ChainConfig{
```

Comment:
That constant fields is not necessary to change. We can start geth and allocate those by config file.

Reply:
We can genesis geth node by genesis.json like this:

{
    "commit": "0000000000000000000000000000000000000000",
    "config": {
        "chainId": 420,
        "homesteadBlock": 0,
        "eip150Block": 0,
        "eip155Block": 0,
        "eip158Block": 0,
        "byzantiumBlock": 0,
        "constantinopleBlock": 0,
        "petersburgBlock": 0,
        "istanbulBlock": 0,
        "muirGlacierBlock": 0,
        "clique": {
            "period": 0,
            "epoch": 30000
        }
    },
   ...
}

Reply:
Correct, though I figured this may be a bit easier since I believe these constant parameters are loaded in if no config.json is provided. So it may help us ensure that we always run with the correct kind of configuration. If you feel it is redundant though, we can simply close the PR.

Reply:
> 

Try not to change if possible, so that others know the difference clearly, and it's convenient for us to upgrade and maintain.

Reply:
Agree with @mask-pp . It'd be good practice to avoid unnecessary changes. We can just add a default genesis config that enables all EIPs.